rfc/drone-ci.md

```diff
@@ -0,0 +1,227 @@
+# Drone CI Spinnaker Stage Type
+| | |
+|-|-|
+| **Status**     | **Proposed**, Accepted, Implemented, Obsolete |
+| **RFC #**      | ***TBD*** |
+| **Author(s)**  | Victor Odusanya (`@Odusanya18`), Cameron Motevasselani (`@link108`) |
```

Comment:
feel free to just have yourself on there, since you were the one to write it, I'm just here to help :)